README.md

```diff
@@ -161,6 +161,22 @@ Next we decide we want the site to be secure all the time, redirecting any traff
 mechanic update mysite --https=true --redirect-to-https=true
 ```
 
+## Redirecting to another site
+
+We can also redirect all traffic to a different site:
+
+```
+mechanic update mysite --redirect=https://example.com
+```
+
+The above redirects 100% of traffic to the same place. To instead append the rest of the original URL to the redirect, use:
```

Comment:
It might be more clear to say, "The above redirects 100% of traffic to the same, exact URL." The skeuomorphic idea of a "place" could be confusing versus the follow option.

Reply:
Suggestion:

> We can also redirect all traffic to a different site. To redirect 100% of the traffic to one specific URL, use `--redirect`
> [EXAMPLE]
> To instead append the rest of the original URL to a redirected domain, use `--redirect-full`.
> [EXAMPLE]